src/java.base/share/classes/java/lang/constant/DirectMethodHandleDesc.java

```diff
@@ -46,6 +46,7 @@
  * {@link MethodHandle}.  A {@linkplain DirectMethodHandleDesc} corresponds to
  * a {@code Constant_MethodHandle_info} entry in the constant pool of a classfile.
  *
+ * @sealedGraph
```

Comment:
DMH desc has no public implementation subtypes. A `@sealedGraph` would probably just render a single root node, which is not informative.

Reply:
Well spotted. Thanks.